UltiSnips/php.snippets

```diff
@@ -8,7 +8,7 @@ $${1:arrayName} = array('${2}' => ${3});${4}
 endsnippet
 
 snippet def "def"
-define('${1}'${2});${3}
+define('${1:FOO}', ${2:'bar'});${3}
```

Comment:
All right. Fixed. I thought FOOs and BARs were pretty straightforward though. =P